real build fail?
```
======================================================================
FAIL: test_app_from_template (corehq.apps.app_manager.tests.test_views.TestTemplateAppViews)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_views.py", line 250, in test_app_from_template
    self._check_response(response)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_views.py", line 241, in _check_response
    self.assertTrue(redirect_location.endswith(expected))
AssertionError: False is not true
```

--- a/corehq/apps/app_manager/tests/test_views.py
+++ b/corehq/apps/app_manager/tests/test_views.py
@@ -237,7 +237,7 @@ class TestTemplateAppViews(TestCase):
         self.assertEqual(response.status_code, 302)
         redirect_location = response['Location']
         [app_id] = re.compile(r'[a-fA-F0-9]{32}').findall(redirect_location)
-        expected = '{}/modules-0/forms-0/source/'.format(app_id)
+        expected = '{}/modules-0/forms-0/'.format(app_id)
         self.assertTrue(redirect_location.endswith(expected))
         self.addCleanup(lambda: Application.get_db().delete_doc(app_id))
 

